### ISSUE TITLE
Incorrect type for validating imagePullSecrets

### ISSUE THREAD

**Issue by gbucknel** [CrowdStrike/falcon-helm#92]
Hello, 

I'm trying to install the daemonset on EKS with a custom docker registry. 
I'm defining the secret like this : 
```
node:
  image:
    repository: "my-registry.com/falcon-node-sensor"
    pullPolicy: Always
    pullSecrets: my-secret
```
Error: values don't meet the specifications of the schema(s) in the following chart(s):
falcon-sensor:
- node.image.pullSecrets: Invalid type. Expected: object, given: string

No matter what format I put in there, it doesn't seem to pass validation. Shouldn't this have 
`-name: <secret name>` ? 

In the actual template it looks like it is just spitting out the secretName, not `-name: my-secret`
https://github.com/CrowdStrike/falcon-helm/blob/main/helm-charts/falcon-sensor/templates/daemonset.yaml#L62

Thanks for any help.

**Comment by gcardonag** [comment]
For any readers, we have switched to using the 1.4.0 release of the chart as a temporary workaround while we wait for a release to include the PR with the schema fix.

**Comment by redhatrises** [comment]
This was fixed in 1.6.1 and is set to change in 1.7.0 shortly to be a string so that all that is required is the name of the secret. Closing this issue.
